BUILD.bazel

```
@@ -0,0 +1,210 @@
+        "include/albatross/src/utils/linalg_utils.hpp",
+        "include/albatross/src/utils/map_utils.hpp",
+        "include/albatross/src/utils/random_utils.hpp",
+        "include/albatross/src/utils/variant_utils.hpp",
+        "include/albatross/src/utils/vector_utils.hpp",
+        "include/albatross/tests/Utils",
+        "include/albatross/utils/AsyncUtils",
+        "include/albatross/utils/CsvUtils",
+        "include/albatross/utils/LinalgUtils",
+        "include/albatross/utils/MapUtils",
+        "include/albatross/utils/RandomUtils",
+        "include/albatross/utils/VariantUtils",
+    ],
+    includes = ["include"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@cereal",
+        "@eigen",
+        "@fast_csv",
+        "@gzip",
+        "@nlopt",
+        "@variant",
+    ],
+)
+
+swift_cc_test(
+    name = "albatorss-test",
+    srcs = [
+        "tests/mock_model.h",
+        "tests/test_apply.cc",
+        "tests/test_async_utils.cc",
+        "tests/test_block_utils.cc",
+        "tests/test_call_trace.cc",
+        "tests/test_callers.cc",
+        "tests/test_concatenate.cc",
+        "tests/test_conditional_gaussian.cc",
+        "tests/test_core_dataset.cc",
+        "tests/test_core_distribution.cc",
+        "tests/test_core_distribution.h",
+        "tests/test_core_model.cc",
+        "tests/test_covariance_function.cc",
+        "tests/test_covariance_functions.cc",
+        "tests/test_covariance_utils.h",
+        "tests/test_cross_validation.cc",
+        "tests/test_csv_utils.cc",
+        "tests/test_distance_metrics.cc",
+        "tests/test_eigen_utils.cc",
+        "tests/test_error_handling.cc",
+        "tests/test_evaluate.cc",
+        "tests/test_gp.cc",
+        "tests/test_group_by.cc",
+        "tests/test_indexing.cc",
+        "tests/test_linalg_utils.cc",
+        "tests/test_map_utils.cc",
+        "tests/test_minimum_spanning_tree.cc",
+        "tests/test_model_adapter.cc",
+        "tests/test_model_metrics.cc",
+        "tests/test_models.cc",
+        "tests/test_models.h",
+        "tests/test_parameter_handling_mixin.cc",
+        "tests/test_patchwork_gp.cc",
+        "tests/test_prediction.cc",
+        "tests/test_radial.cc",
+        "tests/test_random_utils.cc",
+        "tests/test_ransac.cc",
+        "tests/test_samplers.cc",
+        "tests/test_scaling_function.cc",
+        "tests/test_serializable_ldlt.cc",
+        "tests/test_serialize.cc",
+        "tests/test_serialize.h",
+        "tests/test_sparse_gp.cc",
+        "tests/test_stats.cc",
+        "tests/test_traits_cereal.cc",
+        "tests/test_traits_core.cc",
+        "tests/test_traits_covariance_functions.cc",
+        "tests/test_traits_details.cc",
+        "tests/test_traits_evaluation.cc",
+        "tests/test_traits_indexing.cc",
+        "tests/test_tune.cc",
+        "tests/test_utils.h",
+        "tests/test_variant_utils.cc",
+    ],
+    copts = COPTS,
```

```suggestion
    copts = COPTS,
    local_defines ["CSV_IO_NO_THREAD"],
```

This is in the cmake build: https://github.com/swift-nav/albatross/blob/01e266027fa4096165966f1b6b847b4f11944a2d/tests/CMakeLists.txt#L55.

---

Done